packages/medusa/src/types/order-edit.ts

```diff
@@ -21,6 +28,7 @@ export const defaultOrderEditRelations: string[] = [
   "changes.line_item",
   "changes.original_line_item",
   "items",
+  "items.adjustments",
```

Comment:
question: should we return the adjustments here? I think that we never return them unless the client use the expand field

Reply:
These are fine to include :)

Reply:
in that case, @fPolic could I ask you to update the integration tests related to the update line item to not get the order edit with the expand if that is alright, otherwise I ll do it in a separate pr :)

Reply:
ofc, will do